src/components/Banner/Can82Banner.tsx

```diff
@@ -4,15 +4,29 @@ import {
   CondensedBannerWrapperHomepage,
   CondensedBannerWrapperLocationPage,
 } from './Banner.style';
-import { Link } from 'react-router-dom';
+import ExternalLink from 'components/ExternalLink';
+import { EventAction, EventCategory, trackEvent } from 'components/Analytics';
 
-//TODO(8.2) - update with correct learn more URL
+const trackClick = () => {
+  trackEvent(
+    EventCategory.HOMEPAGE_BANNER,
+    EventAction.CLICK_LINK,
+    'Banner Learn More Link',
```

Comment:
The category and action already tell you that the user is clicking a link in the banner - consider changing the label to something more descriptive `Learn Community Levels` or something like that.